sbi/inference/snpe/snpe_base.py

```diff
@@ -277,7 +283,7 @@ def _run_simulations(
 
         return theta, x, self._mask_sims_from_prior(round_, theta.size(0))
 
-    def _z_score_embedding(self, x: Tensor) -> nn.Module:
+    def _prepend_z_score(self, x: Tensor) -> nn.Module:
```

Comment:
This doesn't convince me - sadly I don't have agreat suggestion. `_prepend_standardize` has the only advantage that it is a verb, whereas z_score is a name that can be abused to be a verb (to zscore, much like to google).